openquake/commands/tests/commands_test.py

```diff
@@ -534,11 +538,20 @@ def test_OQ_REDUCE(self):
     def test_sensitivity(self):
         # test the sensitivity of the UHS from the area_source_discretization
         job_ini = os.path.join(os.path.dirname(case_56.__file__), 'job.ini')
-        sap.runline(f'openquake.commands engine --run {job_ini} -c 0')
+        calc = sap.runline(f'openquake.commands engine --run {job_ini} -c 0')
```

Comment:
Please remove `calc` since it is not used

Reply:
Done